e2e/src/addAndDisplayDetails/addAndDisplayDetails.e2e-spec.ts

```diff
@@ -9,12 +9,13 @@ describe('workspace-project App', () => {
   });
 
   it('should display table with 8 action items', () => {
-    page.navigateTo();
+    page.navigateToAllActionItems();
```

Comment:
**FIX** Still tests are using hardcoded ids instead of pattern:
1. Create Action Item.
2. Check if it's displayed on the list.
3. Check if it's displayed in the details etc.